Dockerfile-UI

```diff
@@ -1,12 +1,14 @@
 FROM node:10
 
-RUN mkdir /code
-WORKDIR /code
+RUN mkdir /Frontend /Analytics
+RUN mkdir /Analytics/settings
 # Install dependencies
 # package*.json used to copy files package.json and package-lock.json
-COPY Frontend ./
+COPY /Frontend /Frontend
+COPY /Analytics/settings/config.env.yml /Analytics/settings/config.env.yml
```

Comment:
This line will crash now

Reply:
Unless you give them explicit instructions in your readme/doc on adding the file in there